modules/cbMap/processmap/importCSV.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
```

Comment:
Please add the license here: https://github.com/tsolucio/corebos/blob/master/build/code_license.tpl#L50